WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -143,13 +143,16 @@ public async Task<CoinJoinResult> StartCoinJoinAsync(Func<Task<IEnumerable<Smart
 			currentRoundState = await WaitForRoundAsync(excludeRound, cancellationToken).ConfigureAwait(false);
 			RoundParameters roundParameters = currentRoundState.CoinjoinState.Parameters;
 
-			if (!IsRoundEconomic(roundParameters.MiningFeeRate))
+			if (!currentRoundState.IsBlame)
 			{
-				throw new CoinJoinClientException(CoinjoinError.UneconomicalRound, "Uneconomical round skipped.");
-			}
-			if (SkipFactors.ShouldSkipRoundRandomly(SecureRandom, roundParameters.MiningFeeRate, RoundStatusUpdater.CoinJoinFeeRateMedians, currentRoundState.Id))
-			{
-				throw new CoinJoinClientException(CoinjoinError.RandomlySkippedRound, "Round skipped randomly for better privacy.");
+				if (!IsRoundEconomic(roundParameters.MiningFeeRate))
```

Comment:
Did you check this is the only place it's being used?

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/11795/commits/aded5c09506ec458dedf1df89e78410d400cba20